### DISCUSSION TITLE
lncli timezone?

### DISCUSSION THREAD

**Discussion by ZoltanAB** [lightningnetwork/lnd#6634] [Troubleshooting]
For some reason "lncli fwdinghistory" shows my routings with a 6 hour delay. Both LNDg and RTL shows the forwardings correctly, but when running from terminal "lncli fwdinghistory" I can only see routings with a 6 hour delay.
I think it has something to do with the server timezone or lnd timezone.

Now, the server time is UTC and LND log file also logs in UTC. However, I changed in the past the server time from EEST to UTC and vice versa. Not sure if this caused the problem.

What is strange is thet both LNDg and RTL shows real time data, I thought they also use lncli, so how do they get the data without the delay?

What should I do? Any help is appreciated.

**Comment by yyforyongyu** [comment]
I think `lnd` doesn't manage time zone but relies on the system's time zone. for LNDg and RTL, I'd imagine they are using RPC instead of cli. The difference is, for RPC, you need to construct the unit timestamp yourself, for cli, the timestamp is calculated for you so you can use values like `-200s`, `1d`, etc.

> I changed in the past the server time from EEST to UTC and vice versa. Not sure if this caused the problem.

I think this is the cause. What are the values you used for `lncli fwdinghistory`?
